coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -781,11 +779,18 @@ def flatten(context, node):
         raise ValueError(
             "Start must be before end. (start, end) == ({}, {})".format(start, end_val)
         )
-    for dim in dims[start : end + 1]:
```

Comment:
Please guard this logic when the input shape is static,
you can do that by using `any_symbolic` utility function.

The reason I am asking it is, for the static cases, it would be much more sufficient "not" using `-1` for the `shape` in the `reshape` op.

If we are using the dynamic input shape logic for the static case, that would harm the performance.